source/_integrations/matter.markdown

```diff
@@ -34,13 +34,17 @@ The Matter integration allows you to control Matter devices on your local Wi-Fi
 The integration is marked BETA: Both the Matter standard itself and its implementation within Home Assistant are in a early stage. You may run into compatibility issues and/or other bugs.
 </div>
 
-# What is Matter?
+# Introduction- What is Matter?
 
-Matter is [the new standard for home automation](<https://en.wikipedia.org/wiki/Matter_(standard)>), which has been released recently. It is in the process of being adopted by the tech industry. Matter is a local protocol. Device control is done without the need of any cloud. From a technical perspective, you can use a Matter-compatible device with Home Assistant without connecting to a vendor-specific cloud. However, some vendors may require you to set up an account before you can enable Matter.
+Matter is a new smart home connectivity standard for home automation products and IoT (Internet of Things) devices, see its [Wikipedia article](https://en.wikipedia.org/wiki/Matter_(standard)).
```

Comment:
Hmm, but it is ”a” new standard. There are no garantees that it is ”the” standard for all time. Grammarly it also sounds better to write that it ”a new standard”, at least in my opinion

Reply:
> Hmm, but it is ”a” new standard. There are no garantees that it is ”the” standard for all time. Grammarly it also sounds better to write that it ”a new standard”, at least in my opinion. @marcelveldt Is it not OK if it reads "a" instead of "the"?

@marcelveldt Can I assume that it should be OK if it reads "a" instead of "the"?

Reply:
Well, it is THE standard at this time but that is more something for blogposts and news articles, so yes, fine to leave "s standard" here in documentation.